### PR TITLE
Mention correct shortcut to open "Developer Tools"

### PR DESCRIPTION
**Pull request by koenpunt**
`⌘⌥J` is for "JavaScript Console", `⌘⌥I` is for "Developer Tools"

## Motivation (required)

The shortcut mentioned was for opening the "JavaScript Console", which is incorrect.

image

### CODE REVIEW

File: local-cli/server/util/debugger.html [hunk 1]
```diff
@@ -248,7 +248,7 @@
     <p>
       React Native JS code runs as a web worker inside this tab.
     </p>
-    <p>Press <kbd id="shortcut" class="shortcut">⌘⌥J</kbd> to open Developer Tools. Enable <a href="https://stackoverflow.com/a/17324511/232122" target="_blank">Pause On Caught Exceptions</a> for a better debugging experience.</p>
+    <p>Press <kbd id="shortcut" class="shortcut">⌘⌥I</kbd> to open Developer Tools. Enable <a href="https://stackoverflow.com/a/17324511/232122" target="_blank">Pause On Caught Exceptions</a> for a better debugging experience.</p>
```
Comment (hramos) [inline local-cli/server/util/debugger.html]: Is this the right place to make this edit? I am not familiar with these docs but I'd expect there to be a different source of truth.

Comment (koenpunt) [inline local-cli/server/util/debugger.html]: I'm pretty sure this is the source. However I've seen the incorrect shortcut in a markdown file somewhere else too, so I will update it there also.

Comment (koenpunt) [inline local-cli/server/util/debugger.html]: Oh well, turned out the other place that mentions the devtools already does this correctly:

https://github.com/facebook/react-native/blob/master/docs/Debugging.md#chrome-developer-tools

**Comment by hramos** [comment]
Hi there! This issue is being closed because it has been inactive for a while.

But don't worry, it will live on with Canny! Check out its new home: https://react-native.canny.io/feature-requests/p/mention-correct-shortcut-to-open-developer-tools

**Pull request closed without merging**